apps/console/src/extensions/components/sms-providers/configs/ui.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import  CustomLogo  from "../../../../themes/default/assets/images/third-party/custom-sms-provider.png";
+import TwilioLogo  from "../../../../themes/default/assets/images/third-party/twilio-sms-provider.png";
+import   VonageLogo  from "../../../../themes/default/assets/images/third-party/vonage-sms-provider.png";
+
+export const getSMSProviderIcons = (): {
+    twilio: any;
```

Comment:
Shall we define the the type here as string?

Reply:
fixed